cmd/fetch_k8s_resource.go

```diff
@@ -20,23 +20,15 @@ import (
 	"context"
 	"log"
 
-	appstudioshared "github.com/redhat-appstudio/managed-gitops/appstudio-shared/apis/appstudio.redhat.com/v1alpha1"
+	ecc "github.com/hacbs-contract/enterprise-contract-controller/api/v1alpha1"
 	"github.com/spf13/cobra"
 
-	"github.com/hacbs-contract/ec-cli/internal/applicationsnapshot"
 	"github.com/hacbs-contract/ec-cli/internal/image"
 )
 
 func k8sResourceAuthorizationCmd() *cobra.Command {
 	var data = struct {
-		imageRef  string
-		publicKey string
-		filePath  string
-		input     string
-		namespace string
-		server    string
-		resource  string
-		spec      *appstudioshared.ApplicationSnapshotSpec
```

Comment:
Perhaps there is also value in having this provided individually? Either way is fine really...

Reply:
Possibly, but since I don't have a clear picture of it, maybe we leave it out for now.